app/_src/policies/meshcircuitbreaker.md

```diff
@@ -474,7 +474,35 @@ spec:
 
 #### Basic circuit breaker for outbound traffic from web, to backend service
 
-{% policy_yaml usage use_meshservice=true %}
+{% if_version lte:2.8.x %}
+{% policy_yaml usage-28x use_meshservice=true %}
```

Comment:
If this is lte 2.8 shoudn't it not be using meshservice?

Reply:
removed